src/www/ui/api/documentation/openapi.yaml

```diff
@@ -5663,6 +5697,17 @@ components:
           type: boolean
           example: true
           description: Indicates whether the comment is enabled or not.
+    GetDatabaseMetric:
+      type: object
+      properties:
+        metric:
+          type: string
+          description: The metric associated with the object.
+          example: FOSSology database size
+        total:
+          type: integer
```

Comment:
API returns string instead of integer.

![image](https://github.com/fossology/fossology/assets/18077542/11346f84-e56b-4e6e-a28e-c34595b51b0a)

Reply:
Done @GMishx